src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -64,6 +65,9 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 	useEffect(() => {
 		let isMounted = true;
 		if (value?.length > 0 && isModalOpen) {
+			if (value.length >= fieldDef.inputSettings?.selectLimit) {
+				setCheckboxListDisabled(true)

```

Comment:
Please don't forget the ";" at the end of each line. It might not be as important in javascript but in typescript it can lead to production errors.